internal/user_validator.go

```diff
@@ -88,26 +92,99 @@ func (i *ValidateUser) Setup(ctx context.Context) error {
 	return nil
 }
 
-func decodePgpKey(pgpKey string) (*openpgp.Entity, error) {
+// crc24 calculates the CRC24 checksum for a given byte array.
+//
+// See the RFC 4880, "OpenPGP Message Format", Section 6.1, for source of this implementation.
+func crc24(bytes []byte) uint32 {
```

Comment:
Even though it follows the reference implementation, can you please add a  simple test for this?

Reply:
@janboll, done! Thank you!